package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "tsc --build",
     "cw": "node dist/cold-wallet/main",
-    "cw:dev": "ts-node --files cold-wallet/main.ts",
+    "cw:dev": "ts-node --files cold-wallet/dev-main.ts",
```

Comment:
Better add a new script 😉

Reply:
oh thought that cw:dev is the one i can change. if so, i will add cw:local and mn:local